src/routes/terms/+page.markdoc

```diff
@@ -64,50 +54,55 @@ You agree not to disclose your password to any third party. You must notify us i
 
 The Service and its original content (excluding Content provided by users), features, and functionality are and will remain the exclusive property of Appwrite and its licensors. The Service is protected by copyright, trademark, and other laws of both Israel and foreign countries. Our trademarks and trade dress may not be used in connection with any product or service without the prior written consent of Appwrite.
 
-Notwithstanding any other provision of these Terms, to the maximum extent permitted by law, we may collect, generate, process, use and/or publish Anonymous Information (defined below) relating to your, your users’ use of the Service, and/or disclose it, in order to provide, improve and publicize the Service and our products and services, in order to track general industry trends, develop and publish white papers, reports and summaries, and for other legitimate business purposes, all the foregoing without restriction or limitation of any kind.  “Anonymous Information” means information that does not enable the identification of an individual, such as aggregated and analytics information. For clarity, Anonymous Information will not include any personally identifiable information, and Appwrite shall own all Anonymous Information collected or obtained by Appwrite.
+Notwithstanding any other provision of these Terms, to the maximum extent permitted by law, we may collect, generate, process, use, and/or publish Anonymous Information (defined below) relating to your and your users’ use of the Service, and/or disclose it, in order to provide, improve, and publicize the Service and our products and services, track general industry trends, develop and publish white papers, reports, and summaries, and for other legitimate business purposes, all without restriction or limitation of any kind. "Anonymous Information" means information that does not enable the identification of an individual, such as aggregated and analytics information. For clarity, Anonymous Information will not include any personally identifiable information, and Appwrite shall own all Anonymous Information collected or obtained by Appwrite.
 
-# Prohibited Use of the SMS Messages Service
+# Prohibited Use
 
-You agree not to use the Service for the sending of unsolicited or spam SMS messages. This includes but is not limited to the prohibition of 
-- using the Service in connection with any content, product, or service the recipient has not explicitly requested.
-- sending SMS messages through the Service unless you have obtained prior express consent from the recipients, as required by applicable law.
+You agree not to use the Service for any unlawful or prohibited activities. This includes, but is not limited to:
```

Comment:
The exact contents of prohibited-activities.md is added here so this results in:
<img width="759" alt="image" src="https://github.com/user-attachments/assets/da1ec5ea-eb31-40fc-9151-f140ce2bf0e6" />

As such, we don't need "# Prohibited Use" or "You agree not to use the Service for any unlawful or prohibited activities. This includes, but is not limited to:"